src/gcp_scanner/crawl.py

```diff
@@ -562,9 +558,9 @@ def get_bq(project_id: str,
       response = request.execute()
 
       for dataset in response.get("datasets", []):
-        dataset_id = dataset["datasetReference"]["datasetId"]
-        bq_datasets[dataset_id] = get_bq_tables(project_id, dataset_id, service)
-
+        datasetid = dataset["datasetReference"]["datasetId"]
+        bq_datasets[datasetid] = get_bq_tables(project_id,datasetid, service)
+      
```

Comment:
Trim trailing whitespace

Reply:
I have made the requested changes in the latest PR. Check and let me know if any changes required.